modules/openapi-generator/src/main/resources/typescript-angular/api.service.mustache

```diff
@@ -349,9 +349,13 @@ export class {{classname}} {
 
 {{/hasFormParams}}
     {{^isResponseFile}}
-        let responseType_: 'text' | 'json' = 'json';
-        if(localVarHttpHeaderAcceptSelected && localVarHttpHeaderAcceptSelected.startsWith('text')) {
-            responseType_ = 'text';
+        let responseType_: 'text' | 'json' | 'blob' = 'json';
+        if (localVarHttpHeaderAcceptSelected) {
+            if (localVarHttpHeaderAcceptSelected.startsWith('text')) {
+                responseType_ = 'text';
+            } else if (!localVarHttpHeaderAcceptSelected.includes('json')) {
```

Comment:
I would suggest to use isJsonMime here:

https://github.com/OpenAPITools/openapi-generator/blob/8a147e6a25b603aee512cd30ec99d83db5e4f076/modules/openapi-generator/src/main/resources/typescript-angular/configuration.mustache#L149

Reply:
Sounds good! I'll replace that soon so that we are using common logic to identify JSON.

Reply:
Almost there... I pushed my first attempt at it, then noticed my last changes were never saved to file. Oops. Pushing again soon.

Reply:
After a few mistakes I think I've got this addressed now! Let me know if you'd like me to do anything to cleanup the history of this branch.